server/events/runtime/plan_step_runner_test.go

```diff
@@ -502,14 +502,7 @@ func TestRun_UsesDiffPathForProject(t *testing.T) {
 
 // Test that we format the plan output for better rendering.
 func TestRun_PlanFmt(t *testing.T) {
-	rawOutput := `Refreshing Terraform state in-memory prior to plan...
```

Comment:
I may not be understanding but in order for this change to be backwards compatible shouldn't this test case be left as is and another test case added for the new style output?

Then both test cases should be passing.